prototypes/form-wizard/src/foia.js

```diff
@@ -1,10 +1,50 @@
+// global methods in our namespace
+// AGENCIES is a global var defined in agencies.js
 /* global AGENCIES */
-var FOIA = {
+var FOIA = {};
+
+function Agency(attributes) { Object.assign(this, attributes); }
+
+FOIA = {
   agencyNames: function () {
     return jQuery.map(AGENCIES, function (el, idx) { return idx; }).sort();
+  },
+  agency: function (name) {
+    return new Agency(AGENCIES[name]);
+  },
+  hideAgencyDetails: function () {
+    $('.external-link').hide();
+    $('.unified-form').hide();
+  },
+  showRequestFormLink: function (agency) {
+    var $link = $('<a class="foia-link"></a>');
+    $link.attr('href', agency.request_form);
+    $link.text(agency.request_form);
+    $('p.foia-link').html($link);
+    $('.external-link').show();
+  },
+  showForm: function () { // TODO agency is passed but not used
+    $('.unified-form').show();
   }
 };
 
+// Agency methods
+Agency.prototype.isFOIAOnline = function () {
```

Comment:
I feel like the distinction between what the technology behind the platform is not important, just the fact they have their own request form. I might just collapse the regex matching into `hasRequestForm` and we'll just add additional checks directly there. Do you think the technology distinction is important?